web/src/components/AppFooter.vue

```diff
@@ -5,41 +5,94 @@
     app
     absolute
   >
-    <v-container>
-      <v-row no-gutters>
+    <v-container
+      fluid
+    >
+      <v-row
+        no-gutters
+      >
         <v-col
           cols="12"
-          justify="center"
-          align="center"
+          md="4"
+          xs="12"
         >
-          <a
-            class="footer-link"
-            target="_blank"
-            href="https://github.com/WWCodeTokyo/speak-her-db"
-          >{{ $t('footer.dateString') }}</a>
+          <v-row :justify="$vuetify.breakpoint.mdAndUp ? 'left' : 'center'">
```

Comment:
This is kinda minor but I've seen it in a few files already: can we avoid the unnecessary inner `row` and move the styling to the `col` and parent `row`?

My suggested implementation would be [this one](https://github.com/WWCodeTokyo/speak-her-db/pull/179#discussion_r499583984)

(Saying that cause the standard implementation of the grid system is supposed to be row > col > content, instead of row > col > row > content)

Reply:
I tried your suggestion and was unable to get the layout working. The only way I got it to format the way I want was to use the inner row, so I'd like to leave it as is.

Reply:
Ok, no problem then! I'll take a look later on and submit a PR if I feel the need 👍

Reply:
Sure thing, if you find a way to make it work! I got tired of trying 😆